packages/utilities/src/utils/common.ts

```diff
@@ -109,7 +115,7 @@ export const injectScript = (
             ': ' +
             realSrc +
             ' or global var ' +
-            remoteGlobal +
+            containerKey +
```

Comment:
global shouldnt change to conainter key, only the webpack_require.l needs to change. You still point to the same global regardless

Reply:
Just changed this back